x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_installed_integrations.tsx

```diff
@@ -17,37 +18,18 @@ export interface UseInstalledIntegrationsArgs {
 export const useInstalledIntegrations = ({ packages }: UseInstalledIntegrationsArgs) => {
```

Comment:
I'd move this hook to `x-pack/plugins/security_solution/public/detections/components/rules/related_integrations` to keep it closer to other related code.

Reply:
Addressed in https://github.com/elastic/kibana/pull/133050 👍